content/en/docs/java/manual_instrumentation.md

```diff
@@ -341,16 +352,16 @@ The following is an example of counter usage:
 
 ```java
 // Gets or creates a named meter instance
-Meter meter = OpenTelemetry.getGlobalMeter("instrumentation-library-name","semver:1.0.0");
+Meter meter = meterProvider.get("instrumentation-library-name", "1.0.0");
```

Comment:
Since metrics is still very alpha, I wonder if we should have any documentation about metrics in here at all yet.